classes/class-uagb-init-blocks.php

```diff
@@ -776,6 +776,7 @@ function( $a, $b ) {
 				'current_post_id'                         => get_the_ID(),
 				'btn_inherit_from_theme'                  => UAGB_Admin_Helper::get_admin_settings_option( 'uag_btn_inherit_from_theme', 'disabled' ),
 				'wp_version'                              => get_bloginfo( 'version' ),
+				'is_block_theme'                          => ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ),
```

Comment:
```suggestion
				'is_block_theme'                          => UAGB_Admin_Helper::is_block_theme,
```

i see we are usin this reperative code at multiple places can be replace it with a similar helper function.

Reply:
Done. Please review